docroot/themes/humsci/humsci_basic/src/scss/components/_views-exposed-form.scss

```diff
@@ -12,35 +13,36 @@
       width: calc(50% - #{hb-calculate-rems(18px)});
       margin-right: hb-calculate-rems(18px);
     }
-
-    .hb-layout-builder-main-content & {
-      @include grid-media-min('lg') {
-        width: calc(33% - #{hb-calculate-rems(18px)});
-      }
-    }
-
-    .hb-three-column__sidebar-1 & {
-      @include grid-media-min('lg') {
-        width: 100%;
-      }
-    }
   }
 
   .form-actions {
     display: flex;
     align-items: center;
-    width: auto;
     margin-bottom: 0;
+
     @include grid-media-min('md') {
       margin-bottom: hb-calculate-rems(24px);
     }
   }
 
-  .form-submit {
-    width: auto;
+  .form-submit:not(:last-child) {
+    margin-right: hb-calculate-rems(16px);
+  }
+}
 
-    &:not(:last-child) {
-      margin-right: hb-calculate-rems(16px);
+.hb-layout-builder-main-content .views-exposed-form {
```

Comment:
again, not sure this is the right class to use.. I think `hb-three-column__main` is better in all instances unless there's another reason